### PR TITLE
[type:refactor #3088]format role-sqlmap.xml

### PR DESCRIPTION
**Pull request by Albertsirius**
// Describe your PR here; eg. Fixes #issueNo
format role-sqlmap.xml #3088

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by midnight2104** [comment]
Thanks for your PR.  pls modify name of pr by https://shenyu.apache.org/community/issue-pr . input about #3088 in your comment

File: shenyu-admin/src/main/resources/mappers/role-sqlmap.xml [hunk 1]
```diff
@@ -140,12 +157,12 @@
...
+         WHERE id = #{id,jdbcType=VARCHAR}
     </update>
 
     <delete id="delete" parameterType="java.util.List">
-        delete from role
-        where id in
+        DELETE FROM role
+              WHERE id IN
         <foreach item="id" collection="list" open="(" separator="," close=")">
             #{id,jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/role-sqlmap.xml:167]: good

**Comment by Albertsirius** [comment]
I revised the title and added #3088 in my comment.

**Review by midnight2104 — APPROVED** [review]
good job. If you want, you can add me on WeChat `LL332595980` and invite you to join the shenyu contributor group.

**Pull request merged**